### PR TITLE
PLAT-129909: Fixed dropdown being closed when click up/down buttons inside scroller

### PR DESCRIPTION
**Pull request by daniel-stoian-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A CHANGELOG entry is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Issues: 
agate/Dropdown: The dropdown is closing when clicking the up/down buttons of the DropdownList scroller

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Solution: 
removed on the call to onClose when the dropdownButton is clicked

### Links
[//]: # (Related issues, references)
PLAT-129909

### Comments

Enact-DCO-1.0-Signed-off-by: Daniel Stoian daniel.stoian@lgepartner.com

### CODE REVIEW

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM

**Review by stanca-pop-lgp — APPROVED** [review]

File: Dropdown/Dropdown.js [hunk 1]
```diff
@@ -191,6 +191,9 @@ const DropdownBase = kind({
 		onOpen: handle(
 			forProp('open', false),
 			forward('onOpen')
+		),
+		onScrollerClick: handle(
```
Comment (0x64) [inline Dropdown/Dropdown.js:195]: Just note: actually we don't need `handle` for this as we don't use any props and no chaining functions here.  But it looks not bad to me.

**Comment by 0x64** [comment]
:lgtm:

**Review by seunghoh — APPROVED** [review]
LGTM!

**Pull request merged**